Source/MediaInfo/MediaInfo_Config_MediaInfo.cpp

```diff
@@ -2995,19 +2996,18 @@ float64 MediaInfo_Config_MediaInfo::File_GrowingFile_Delay_Get ()
 }
 
 //---------------------------------------------------------------------------
-void MediaInfo_Config_MediaInfo::File_GrowingFile_Force_Set (float64 NewValue)
+void MediaInfo_Config_MediaInfo::File_GrowingFile_Force_Set (bool NewValue)
 {
     CriticalSectionLocker CSL(CS);
-    if (NewValue)
-    {
-        File_IsGrowing=true;
-        File_IsNotGrowingAnymore=false;
-    }
-    else
-    {
-        File_IsGrowing=false;
-        File_IsNotGrowingAnymore=true;
-    }
+    File_GrowingFile_Force=NewValue;
+}
+
+//---------------------------------------------------------------------------
+bool MediaInfo_Config_MediaInfo::File_GrowingFile_Force_Get ()
+{
+    CriticalSectionLocker CSL(CS);
+    float64 Temp=File_GrowingFile_Force;

```

Comment:
hum... bad habits, from copy/paste.
was used before CriticalSectionLocker is there, definitely useless since there is a CriticalSectionLocker making the temp value no more needed.

Good point, but I would like to do a complete review of the .cpp removing the temp value everywhere, so I keep it as is for this PR and plan for fixing that everywhere in the file with another PR (me... or you if you are motivated)